### ISSUE TITLE
Kubernetes server minimum version supported not enforced in viya_deployment_settings.ini

### ISSUE THREAD

**Issue by raphaelpoumarede** [sassoftware/viya4-ark#203]
Hello

I'm using version 2.0.0 and the release notes says:
"Kubernetes server minimum version supported: 1.24.0."

However in /viya4-ark/pre_install_report/viya_deployment_settings.ini I see:
```sh
# Minimum Kubernetes Server Version supported.  Format: major.minor
VIYA_K8S_VERSION_MIN=1.21
```
is it expected ?

**Comment by lasiva** [comment]
Hi Raphael.  I think the tools needs to be updated.  Let me get back to you after I have examined the message. Thanks!

**Comment by kevinlinglesas** [comment]
This issue is addressed in [Release 2.0.1](https://github.com/sassoftware/viya4-ark/releases/tag/2.0.1)
